CHANGELOG.md

```diff
@@ -30,16 +41,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 ### Removed
 * Remove UNET [MTT-3435] (#638) --> removed deprecated UNET transport from Boss Room
 
-## [Unreleased] - yyyy-mm-dd
 
-### Added
-*
-### Changed
-*
-### Removed
-*
-### Fixed
```

Comment:
shouldn't that line stay here?

Reply:
nvm I misread this, on my side it looked like that change was in the section below